src/layouts/shortcodes/cloud-prepare/aws.md

```diff
@@ -0,0 +1,54 @@
+<!-- Hugo doesn't allow shortcodes in shortcodes, so use HTML -->
+<div class="notices note">
+<p>The default EC2 instance type for the Submariner gateway node is <code>m5n.large</code>,
+optimized for improved network throughput and packet rate performance.
+Please ensure that the AWS Region you deploy to supports this instance type.
+Alternatively, you can choose to deploy using a different instance type.</p>
+</div>
+
+<!-- Hugo doesn't allow shortcodes in shortcodes, so use HTML -->
+<div class="notices warning">
+<p>The following permissions need to be enabled in IAM on the <code>openshift-machine-api</code> user for your cluster:</p>
+<ul>
+<li>CreateTags</li>
+<li>DeleteTags</li>
+<li>CreateSecurityGroup</li>
+<li>DeleteSecurityGroup</li>
+<li>AuthorizeSecurityGroupIngress</li>
+<li>RevokeSecurityGroupIngress</li>
+<li>DescribeInstanceTypeOfferings</li>
+</ul>
+</div>
+
+You need to extract your infra ID and region in order to use the command.
+Its possible to extract them from the `metadata.json` file that the OpenShift Installer created:
+
+```bash
+metadata_file=<path/to>/metadata.json
+infra_id=$(jq -r .infraID $metadata_file)
+region=$(jq -r .aws.region $metadata_file)
+```
+
+{{- $clusters := $.Get "clusters" }}
+{{- range $cluster := split $clusters "," }}
+Run the command for **{{ $cluster }}**:
+
+```bash
+export KUBECONFIG={{ $cluster }}/auth/kubeconfig
+subctl cloud prepare aws --infra-id $infra_id --region $region {{ with $.Get "nattPort" }}--natt-port {{.}}{{ end }}
+```
+
+{{- end }}
+
+Note that certain parameters, such as the IPsec UDP ports and AWS instance type for the gateway,
```

Comment:
```suggestion
Note that certain parameters, such as the tunnel UDP port and AWS instance type for the gateway,
```

Reply:
@mkolesnik @mangelajo I suggest to change it to something more generic to also accommodate for WireGuard, VXLAN, and so on.